src/hardware/serial_terminal.rs

```diff
@@ -0,0 +1,77 @@
+use super::UsbBus;
+
+static OUTPUT_BUFFER: bbqueue::BBBuffer<1024> = bbqueue::BBBuffer::new();
```

Comment:
let's start with 512

Reply:
Fixed in 2bd057c35fc9230293f612668f01fc5aff581371